src/lib/styles/global/link.scss

```diff
@@ -16,7 +16,13 @@ a {
   }
 
   &.button {
-    @include button.base;
+    &.primary,
+    &.secondary,
+    &.success,
+    &.danger,
+    &.warning {
+      @include button.base;
```

Comment:
Same as for button to avoid extra paddings on `a.button.text`

Reply:
Looks like currently there is only one place where it's used, so not a breaking change `<a class="button primary full-width with-icon"...`.